src/components/package.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import Loader from './loader'
+import Image from '../images/logo-visit.png'
+import Category from './category'
+import Package from './package_id'
+import { Switch, BrowserRouter as Router, Route, Link } from 'react-router-dom'
+import Error from './error'
+
+class Fetch extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      pkg: [],
+      isloading: true,
+      error: null,
+      category: [],
+    }
+  }
+
+  componentWillMount() {
+    fetch('http://127.0.0.1:3000/api/v1/package/all')
+      .then(res => res.json())
+      .then(pkg_output => {
+        this.setState({ pkg: pkg_output, isloading: false })
+        console.log(pkg_output)
+      })
+      .catch(error => this.setState({ error, isloading: false }))
+
+    fetch('http://127.0.0.1:3000/api/v1/category/all')
+      .then(res => res.json())
+      .then(category_output => {
+        this.setState({ category: category_output, isloading: false })
```

Comment:
**IMPORTANT**

Not the right way to execute two fetch requests.
First call package url, and then once you got the output in `then`, call fetch category url inside it. Once you have both categories and packages, then set all the states.